safe/impact_statistics/function_options_dialog.py

```diff
@@ -236,27 +237,36 @@ def build_widget(self, form_layout, name, key_value):
         hide_label = False
 
         if isinstance(key_value, list):
+            def function(z):
+                """
+                :param z: the values to be cast as csv
```

Comment:
:param z: The values to be cast as csv.
:type z: str    <-- comman separated list of whatever values are allowed.